packages/bodiless-core/src/withSidecarNodes.tsx

```diff
@@ -26,11 +26,17 @@ const SidecarNodeContext = createContext<ContentNode<any>[]>([]);
  * @see `withSidecarNodes`
  *
  * @param Component Any component which uses the Bodiless ContentNode system.
- */
+*/
 const startSidecarNodes = <P extends object>(Component: ComponentType<P>) => {
   const StartSidecarNodes = (props: P) => {
     const oldValue = useContext(SidecarNodeContext);
-    const newValue = [...oldValue, useNode().node];
+    const newValue = [
```

Comment:
I think this is the right approach, but I think we can simplify it (and make it more generic in case other collections are added) by exporting [the actual context itself](https://github.com/dewen/Bodiless-JS/blob/feat/implement-interface-for-managing-meta-data/packages/bodiless-core/src/NodeProvider.tsx#L25) from `NodeProvider` and just saving/restoring its current value.  So, in NodeProvider

Reply:
Make sure you export the type (NodeMap) also.  And don['t export from the package...